### ISSUE TITLE
Angled panels do not allow one to enter a portal on them

### ISSUE THREAD

**Issue by n1c9** [BEEmod/BEE2-items#1748]
I'm sorry if this has been mentioned already, but I am trying to make a map that requires players to shoot successive portals into angled panels to keep them inflight, and it appears as though it is impossible to enter through a portal located on an angle panel. I've tested it with goo, and it is certainly able to enter/exit portals on deployed angled panels, but the player cannot enter one, only exit.

**Comment by LautaroL20** [comment]
Are these panels static and from 30° or 45° Degrees?

**Comment by n1c9** [comment]
Yes, I only tested them so static with no inputs at 45 and 30

**Comment by LautaroL20** [comment]
One more thing though, on wich style does this issue occurs? I heard it happens in Clean Style but not sure if it goes for others aswell.

**Comment by LautaroL20** [comment]
Yep, this does happens in Overgrown aswell, ill see if i can fix them in Hammer. 

Also, this is a duplicate of: https://github.com/BEEmod/BEE2-items/issues/1438

**Comment by LautaroL20** [comment]
@n1c9 - Ive figured out why this issue happens. And it will be fixed for next pre-release 25.

**Comment by TeamSpen210** [comment]
Why is it?

**Comment by TeamSpen210** [comment]
Be very careful, I couldn't find a good combination of clips.

**Comment by LautaroL20** [comment]
The `func_brush` solidity of each static instances must be set to "Never Solid" so it will allow players to join them trhough portals without issues.

**Comment by TeamSpen210** [comment]
Nope!

**Comment by LautaroL20** [comment]
But they are pretty much solid, im not a guy who don´t playtest things well before commiting! Want some screenshots?

**Comment by TeamSpen210** [comment]
Make sure to test all these things:
* Turret fire and portal shots
* line-of-sight for turrets
* physics objects
* players
* gel
* light brIdges
* lasers
* funnels
* Sendificator

It needs to block all of these, but also not block light (otherwise you get weird shadows). For that reason it can't be a world brush or func_detail, as well because they can delete the actual visible brushes.

**Comment by LautaroL20** [comment]
Ive tested with Funnels & Light Bridges and Static Panels still reacts pretty much solid to them, also for Cubes. Ill have to playtest rest of the things.

**Comment by TeamSpen210** [comment]
Those are big enough that they'll be blocked by the rest of the geometry. Stuff like lasers or portal shots are more important, they can be precisely targeted.

**Comment by LautaroL20** [comment]
Good news, it works all of that you mentioned adobe. Here is an screenshot in case you want to see results:

![20170316093801_1](https://cloud.githubusercontent.com/assets/12244376/23997203/23bf2b16-0a2f-11e7-8445-8170f821c86e.jpg)

**Comment by LautaroL20** [comment]
It blocks* Also for some reason the sendificator in P1 Style actually doesn´t sendificate anything..... Ill report this bug.

**Comment by TeamSpen210** [comment]
No, it doesn't, do it in the other direction. You can see it goes straight through the clip brush.

**Comment by LautaroL20** [comment]
Okay, this one screen is what i call "All in one", it literally blocks everything:
![Image](https://cloud.githubusercontent.com/assets/12244376/24037058/13496966-0adb-11e7-9813-e7c00711c3cb.jpg)

**Comment by TeamSpen210** [comment]
That's the wrong side. The backside also needs to block stuff, the area where the panel arms would be. I know the front blocks it, but not the nodraw clip. Move the lasers so they're firing in the other horizontal direction.

**Comment by LautaroL20** [comment]
![Image](https://cloud.githubusercontent.com/assets/12244376/24038069/b46a637e-0ade-11e7-97f0-5505cf474a3b.jpg)

**Comment by TeamSpen210** [comment]
I meant 90° around, but that also works. You can see the cube's inside the clip space, it's not doing anything at all.

**Comment by LautaroL20** [comment]
Why there should be a mysterious invisible clip that blocks stuff in an unrealistic way there? I mean, front and back side Panel´s Geometry and its Piston model should be the only things capable from blocking players and stuff.

**Comment by TeamSpen210** [comment]
They need to match the other styles, so switching styles won't break puzzles.

**Comment by LautaroL20** [comment]
Clean:
![20170317084801_1](https://cloud.githubusercontent.com/assets/12244376/24042894/c7d84eee-0af2-11e7-82b7-92cbba2591a6.jpg)

Overgrown: 
![20170317084135_1](https://cloud.githubusercontent.com/assets/12244376/24042856/9f031d14-0af2-11e7-95a9-edc2a1b5a51e.jpg)

BTS:
![20170317085536_1](https://cloud.githubusercontent.com/assets/12244376/24042951/0a033248-0af3-11e7-9fe6-6b01023b70af.jpg)

50´s and 60´s:
![20170317090340_1](https://cloud.githubusercontent.com/assets/12244376/24042962/1afbea2c-0af3-11e7-9239-89f15248a480.jpg)

70´s:
![20170317090903_1](https://cloud.githubusercontent.com/assets/12244376/24042979/2be57da8-0af3-11e7-9d8e-d66bbfe3c349.jpg)

80´s:
![20170317091718_1](https://cloud.githubusercontent.com/assets/12244376/24043003/3cb4bd38-0af3-11e7-8068-f3105e0963fc.jpg)

**Comment by TeamSpen210** [comment]
Nope, those are very broken, you can see them going straight through the arms.

**Comment by ZombiePwner1** [comment]
@TeamSpen210 I told lautaro what u mean't by they need to match other styles. Guess i messed it up.

**Comment by LautaroL20** [comment]
@TeamSpen210 - I figured a way out to not wasting entities and fix what you mean, ill have to test it though.

**Comment by LautaroL20** [comment]
Done, i just need to re-fix the Retro ones, but here is a screenshot:
![20170317124929_1](https://cloud.githubusercontent.com/assets/12244376/24056818/e3e7055e-0b23-11e7-99e2-09aeac6098d2.jpg)

**Comment by TeamSpen210** [comment]
Looks good. What did you use?

**Comment by LautaroL20** [comment]
I used the clipping tool. Old Aperture ones are fixed aswell now.

**Comment by TeamSpen210** [comment]
No, I mean what combination of materials and ents?

**Comment by LautaroL20** [comment]
Ive re-edited the original nodraw clip that they had in the first place. At first i thought those were stuff that the compiler used to implant their textures, so this was confusing me a bit, but then i understood that they don´t have anything to do with compiler textures and were used to stop things going through here.

**Comment by TeamSpen210** [comment]
The compiler doesn't edit any of the instances, or even open. It only edits templates, saved in the packages. Those are all merged into one big VMF it reads in.

**Comment by TeamSpen210** [comment]
Unfortunately this doesn't work completely. If the portal is placed halfway across two panels, it still blocks you from entering.

**Comment by LautaroL20** [comment]
:/

**Comment by LautaroL20** [comment]
This is a duplicate of: https://github.com/BEEmod/BEE2-items/issues/1438

**Comment by Endermage77** [comment]
Now this is happening again.
Tested with 50's style.

**Comment by TeamSpen210** [comment]
In PR25? Does it happen in 
Clean? Other styles?

**Comment by TeamSpen210** [comment]
What specifically isn't working?

**Comment by Endermage77** [comment]
I have not tested other styles, but at least, in PR25, in 50's style: static panels, on the wall, for 45 degrees (Which is what I have discovered, inadvertently, so far), it is not possible to enter the portal. I'll test other orientations and update as per necessary.

**Comment by Endermage77** [comment]
50's Style:
Not enterable: 60 degree, 45 degree, and 30 degrees, regardless of orientation.
90 degree panels are unaffected.
